packages/console/src/pages/UserDetails/components/CreateSuccess.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { toast } from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import ReactModal from 'react-modal';
+import { useSearchParams } from 'react-router-dom';
+
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import Eye from '@/icons/Eye';
+import * as modalStyles from '@/scss/modal.module.scss';
+
+import * as styles from './CreateSuccess.module.scss';
+
+type Props = {
+  username: string;
+};
+
+const CreateSuccess = ({ username }: Props) => {
+  const [searchParameters, setSearchParameters] = useSearchParams();
+  const [passwordVisible, setPasswordVisible] = useState(false);
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  const passwordEncoded = searchParameters.get('password');
+  const password = passwordEncoded && atob(passwordEncoded);
+
+  const handleClose = () => {
+    setSearchParameters({});
+  };
```

Comment:
useCallback?

Reply:
currently we don't use `useCallback` much, and this function here is not special, may be we can disscuss this?